### ISSUE TITLE
CLN: Switch notebook read/write to use a dedicated reader/writer for each folder

### ISSUE THREAD

**Issue by Jython1415** [Jython1415/zoll-foundation-grant-project#41]
- [x] Blocked by https://github.com/Jython1415/zolltools/issues/92

## Description

Once zolltools has deprecated the "tmp" folder, all notebooks should transition to having a dedicated reader/writer for each folder of tables.

**Comment by Jython1415** [comment]
Major progress has been made in this regard. These new changes can be merged into other development branches, but not info main because another review is required before that.

**Comment by Jython1415** [comment]
^ nevermind. The above is a bad idea. I should focus on completing the two features separately before merging them into main.

**Comment by Jython1415** [comment]
Since the zolltools issue can be resolved, the notebooks in this projects can be updated.

**Comment by Jython1415** [comment]
Done.
